.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 📢 Feedback
+    about: Have an idea for a new feature? Open a discussion and let's talk about it!
+    url: https://github.com/puppetlabs/pdkgo/discussions/new
+  - name: ❔ Ask a question about Puppet Content Templates
+    about: Please check out the GitHub community forum for discussions about the GitHub API.
```

Is this part of the default template?
```suggestion
    about: Please check out the GitHub community forum for discussions about the GitHub API.
```

---

Good catch, copypasta error!